shared/src/business/utilities/pdfGenerator/components/SelectedFiltersSection.tsx

```diff
@@ -1,26 +1,34 @@
 import React from 'react';
 
-export const SelectedFiltersSection = ({ count, selectedFilters }) => (
```

Comment:
Heads-up: I think this only affects half of the code. This screen shot below is for `WorkingCopySessionList`, and I suspect that needs to be updated as well. Easy to miss. Our frontend code has a lot of similar and potentially redundant components.

<img width="1106" alt="Screenshot 2025-06-24 at 10 09 08 AM" src="https://github.com/user-attachments/assets/88c230a0-035e-487b-8dfa-9846e2624d04" />